bin/create_hugo_yaml.py

```diff
@@ -90,7 +91,7 @@ def export_anthology(anthology, outdir, clean=False, dryrun=False):
         name = anthology.people.get_canonical_name(id_)
         log.debug("export_anthology: processing person '{}'".format(repr(name)))
         data = name.as_dict()
-        data["slug"] = id_
+        data["slug"] = slugify(repr(name)) or "NONE"
```

Comment:
There is a `PersonName.slug()` function in bin/anthology/people.py, I think all slug generation should be done in that class rather than here in the build script.